bacom-blog/figure/images-to-figure.js

```diff
@@ -0,0 +1,17 @@
+import { selectAllBlocks } from '../../utils/mdast-utils.js';
+import { select } from 'unist-util-select'
```

Comment:
Lets write a ticket to see what we can refactor using this library after the migration. We may not get to it, but there will be different styles of scripts that we may want to refactor into similar patterns later on.